keyboards/handwired/daskeyboard/daskeyboard4/info.json

```diff
@@ -0,0 +1,158 @@
+{
+    "manufacturer": "Metadot",
+    "keyboard_name": "daskeyboard4",
+    "maintainer": "Commander1024",
+    "url": "https://www.daskeyboard.com/de/daskeyboard-4-professional/",
+    "bootloader": "tinyuf2",
+    "bootloader_instructions": "Hold ESC while plugging USB in",
+    "diode_direction": "ROW2COL",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "encoder": true
+    },
+    "matrix_pins": {
+        "cols": ["C14", "C15", "A0", "A2", "A4", "A5", "A6", "A7", "B0", "B1", "B10", "B9", "B7", "B6", "B5", "B4", "A8"],
+        "rows": ["A13", "A14", "A1", "A3", "B8", "B3", "A15", "B15"]
+    },
+    "board": "BLACKPILL_STM32_F411",
+    "processor": "STM32F411",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0140",
+        "vid": "0x24F0"
```

Comment:
To be blunt, I'm not really comfortable using Das Keyboard's VID here, even if that is what the board was originally. It's been modded with a replacement controller, at the very least.

It should be changed (either to `0xFEED` or something above `0x4000`).

Reply:
Actually I am completely with you and I had left it with the default VID/PID values. Changing it to different values was a recommendation from the discord channel. I asked there earlier, if the branch was PR'able. I'll dig into the topic and choose new ones.